referenceqvm/unitary_generator.py

```diff
@@ -355,6 +356,45 @@ def tensor_gates(gate_set, defgate_set, pyquil_gate, num_qubits):
     return gate
 
 
+def tensor_up(pauli_terms, num_qubits):
+    """
+    Takes a PauliSum object along with a total number of
+    qubits and returns a matrix corresponding the tensor representation of the
+    object.
+
+    Useful for generating the full Hamiltonian after a particular fermion to
```

Comment:
Doesn't clarify the use case. If you say that then you should give an example

Reply:
yup.